tests/backstop/scenarios.js

```diff
@@ -1,8 +1,84 @@
 const storybookHost = require( './detect-storybook-host' );
 const rootURL = `${ storybookHost }iframe.html?id=`;
 const storybookStories = require( '../../.storybook/storybook-data' );
```

Comment:
It might be a good idea to rename this too as it's a bit confusing. How about:
```suggestion
const legacyStorybookScenarios = require( '../../.storybook/storybook-data' );
```